src/Column/MoneyColumn.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * Symfony DataTables Bundle
+ * (c) Omines Internetbureau B.V. - https://omines.nl/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Omines\DataTablesBundle\Column;
+
+use Omines\DataTablesBundle\DataTable;
+use Omines\DataTablesBundle\Exception\MissingDependencyException;
+use Symfony\Component\Form\Extension\Core\DataTransformer\MoneyToLocalizedStringTransformer;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * MoneyColumn.
+ *
+ * @author Max Mishyn <max.mishyn@wisetiger.co.uk>
+ */
+class MoneyColumn extends AbstractColumn
```

Comment:
It makes more sense to derive from TextColumn I'd say, it gives you the search and raw functionality for free.

Reply:
I though about it. But I used already existing NumberColumn which is derived from AbstractColun as a reference. So I followed this way to maintain consistancy. ;)

Reply:
Yes it makes sense except that MoneyColumn inherently renders HTML as a result, and in doing so may require certain formatting (accountancy layout, &euro; entities. It is essentially an extended text column, as witnessed by the `raw` flag now duplicated.